apps/coscrad-frontend-e2e/src/e2e/resource-index-to-detail-flows/create-note-about-term.e2e.cy.ts

```diff
@@ -0,0 +1,109 @@
+describe(`term detail view `, () => {
```

Comment:
How does this relate to `create-note.e2e.cy.ts`?

Reply:
This is a much more thorough test. I think this should be our single, really strict test of the behaviour and we can add sanity checks for other resource types.